hospital-service/src/main/resources/application-prod.properties

```diff
@@ -0,0 +1,9 @@
+################## Data Source Properties for MYSQL. Comment out if you wish to use another DB ##################
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.datasource.url = jdbc:mysql://${HOSPITAL_DB_URL}:${HOSPITAL_DB_PORT}/microservices_doctors?createDatabaseIfNotExist=true&amp&useSSL=false
+spring.datasource.username = ${HOSPITAL_DB_USERNAME}
+spring.datasource.password = ${HOSPITAL_DB_PASSWORD}
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5InnoDBDialect
+spring.jpa.properties.hibernate.format_sql = TRUE
+spring.jpa.properties.hibernate.show_sql = TRUE
+spring.jpa.hibernate.ddl-auto = update
```

Comment:
add:
spring.jpa.database=MYSQL
without it, derby will get loaded automatically by spring context

Reply:
Pretty sure, right now, even if you run with prod profile, hospital service will still use embedded derby